kernel/term_typing.ml

```diff
@@ -256,10 +256,42 @@ let infer_declaration (type a) ~(trust : a trust) env (dcl : a constant_entry) =
         cook_context = ctx;
       }
 
+  (** We suppose that Primitives are never universe polymorphic *)
```

Comment:
You should enforce it instead of supposing it.

Reply:
Right, what is the recommended way of doing that?